spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/config/DiscoveryClientOptionalArgsConfiguration.java

```diff
@@ -117,7 +114,7 @@ private static void setupTLS(AbstractDiscoveryClientOptionalArgs<?> args, TlsPro
 	}
 
 	@Configuration(proxyBeanMethods = false)
-	@Conditional(JerseyClientPresentAndEnabledCondition.class)
+	@Conditional(RestTemplateEnabledCondition.class)
 	@ConditionalOnBean(value = AbstractDiscoveryClientOptionalArgs.class, search = SearchStrategy.CURRENT)
```

Comment:
We still want to keep the `JerseyClientNotPresentOrNotEnabledCondition` here, whether or not `RestTemplate` is enabled.

Reply:
Yes, I edited this incorrectly 👍